### ISSUE TITLE
playground returns wrong domain in next attribute during pagination

### ISSUE THREAD

**Issue by JirkaChadima** [windingtree/wt-read-api#156]
Request to 
`https://playground-api.windingtree.com/hotels?fields=id,name,description,location,images&limit=2` returns 
```json
{
   "items":[
      {
         "name":"Guest House One Eyed Badger",
         "description":"Small, private guest house in the quiet part of Dragolm. No kids, no pets. Smokers are welcome.",
         "location":{
            "latitude":46.4528031,
            "longitude":23.5229753
         },
         "images":[
            "https://swarm-gateways.net:443/bzz-raw:/8223315e04f9ee3ee9fb2c9e6d44dd92a60407bbc6d19430cb51595bf73d9fb0",
            "https://swarm-gateways.net:443/bzz-raw:/7e63a0e8fe1b13b7cac4d727f1159e0da5dde877a2036d59ade2cc21304e1a68",
            "https://swarm-gateways.net:443/bzz-raw:/410fb2e98bbd2fbbba50c4dce865e17731cb72f94d4f798579ae37392f40e720",
            "https://swarm-gateways.net:443/bzz-raw:/1f9403308dc3d8db27285b7fe1d84695370b91968c935a6dbb4b43ad5d7f251a"
         ],
         "id":"0x50034977Cb8B3ca62b8CCbE87fcD32Ff9170B782"
      },
      {
         "name":"Hostel Marilena",
         "description":"Cheap accomodation in Dragolm for anyone. Good connection with city centre via public transport. Cyclicsts, backpackers welcome. Reception is open 24/7.",
         "location":{
            "latitude":45.9348412,
            "longitude":24.012348
         },
         "images":[
            "https://swarm-gateways.net:443/bzz-raw:/99891ac02104e59fdce6dfcc42de80585539eaa2417bb0f01c0bbf73cb5519c3",
            "https://swarm-gateways.net:443/bzz-raw:/3c3b2aa9e4f8031e0596255854997b8df615f7d132f8f6163d6846215183237c",
            "https://swarm-gateways.net:443/bzz-raw:/27abc31e4b58a83173c0e9b9dfcb608d9c601a5a32c9fe21b605addf47574ef1"
         ],
         "id":"0xD2De5B31CaAd0E51093cA16ae07f64B6BE8E8653"
      }
   ],
   "errors":[

   ],
   "next":"https://demo-api.windingtree.com/hotels?limit=2&fields=id,name,description,location,images&startWith=0xB473DE8099EF1D198fc2d7dfC5376d47926003D0"
}
```

https://github.com/windingtree/wt-read-api/blob/master/src/controllers/hotels.js#L2 is not working for some reason.

**Comment by JirkaChadima** [comment]
Probably fixed with #147 
Re-check after new version gets deployed.

**Comment by JirkaChadima** [comment]
Fixed.
